gm4_calling_bell/data/gm4_calling_bell/functions/trader/ray.mcfunction

```diff
@@ -0,0 +1,6 @@
+# @s = marker used to detect bell
+# run from gm4_calling_bell:trader/init and recursion
+
+scoreboard players add $ray gm4_calling_bell 1
+tp @s ^ ^ ^0.01
```

Comment:
This is the old style of ray casting; an entity is no longer needed. Refer to Note Block Interface for the proper ray casting code